pyproject.toml

```diff
@@ -0,0 +1,12 @@
+[project]
+name = "ndif"
+version = "0.1.0"
+description = "Add your description here"
```

Comment:
Maybe just paste in "The NDIF server, which performs deep inference and serves nnsight requests remotely"? (we probably need a better repo description lol)